core/src/test/java/tech/pegasys/eth2signer/core/multikey/metadata/ArtifactSignerFactoryTest.java

```diff
@@ -39,14 +42,23 @@
   private Path passwordFile;
   private ArtifactSignerFactory artifactSignerFactory;
 
+  final Vertx vertx = Vertx.vertx();
```

Comment:
shouldln't this be in `@BeforeEach`? as you are closing it in `@AfterEach`?

Reply:
done